pkgs/applications/terminal-emulators/wezterm/default.nix

```
@@ -0,0 +1,87 @@
+{ rustPlatform
+, lib
+, fetchFromGitHub
+
+, pkgconfig
+, fontconfig
+, python3
+, openssl
+, perl
+
+, dbus
+, libX11
+, xcbutil
+, libxcb
+, xcbutilkeysyms
+, xcbutilwm # contains xcb-ewmh among others
+, libxkbcommon
+, libglvnd # libEGL.so.1
+, egl-wayland
+, wayland
+, libGLU
+, libGL
+, freetype
+, zlib
+}:
+let
+  runtimeDeps = [
+    libX11
+    xcbutil
+    libxcb
+    xcbutilkeysyms
+    xcbutilwm
+    libxkbcommon
+    dbus
+    libglvnd
+    zlib
+    egl-wayland
+    wayland
+    libGLU
+    libGL
+    fontconfig
+    freetype
+  ];
+
+  pname = "wezterm";
+  version = "3bd8d8c84591f4d015ff9a47ddb478e55c231fda";
```

Can you please tag a release tag.

---

I can't but I can certainly ask @wez to do so :-)

---

soonish; there are a few things slightly in flux in master right now!

---

```suggestion
  version = "unstable-2020-11-22";
```
as prescribed in https://nixos.org/manual/nixpkgs/stable/#sec-package-naming